releases/v1.3.7.toml

```diff
@@ -0,0 +1,29 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.3.6"
+
+pre_release = false
+
+preface = """\
+The seventh patch release for `containerd` 1.3 includes bug fixes related to clean-up of containers that don't exist, suspend a devmapper device to avoid data corruption when taking snapshots, moves the shim context to prevent unexpected timeouts, allow empty scopes for token authorization, and replace ocispec.MediaTypeImageManifest with manifest.MediaType to prevent 400 on ctr push.
```

Comment:
This can be shortened a bit. Normally here is where we highlight the most important fix/fixes which necessitate the release. All the other parts are still mentioned below. The first two are probably enough to mention in this case.